src/home/buttons/camera-button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid } from '@material-ui/core'
+
+function cameras() {
+  window.location.href = '/cameras'
```

Comment:
Also, this method of redirecting is extremely crude. In general, we should try to avoid using the `browser` API, as it can sometimes be ...finicky. [Here's](https://reactrouter.com/web/guides/quick-start) a link to the documentation of `react-router`, where you'll find a much better way of handling redirects.

Reply:
done